packages/monaco/src/browser/monaco-editor-provider.ts

```diff
@@ -63,41 +62,32 @@ export class MonacoEditorProvider {
         @inject(ApplicationServer) protected readonly applicationServer: ApplicationServer,
         @inject(monaco.contextKeyService.ContextKeyService) protected readonly contextKeyService: monaco.contextKeyService.ContextKeyService
     ) {
-        const init = monaco.services.StaticServices.init.bind(monaco.services.StaticServices);
+        const staticServices = monaco.services.StaticServices;
+        const init = staticServices.init.bind(monaco.services.StaticServices);
         this.applicationServer.getBackendOS().then(os => {
             this.isWindowsBackend = os === OS.Type.Windows;
         });
+        if ('resourcePropertiesService' in staticServices) {
```

Comment:
it would be good to expose it explicitly in `monaco.d.ts`, otherwise we won't check its existence during next monaco migration.

Reply:
done